### PR TITLE
feat: allow multiple classes to be set from windowClass

### PR DESCRIPTION
**Pull request by fschick**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
When two classes added to `DialogConfig.windowClass` a
```DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('center dialog-lg') contains HTML space characters, which are not valid in tokens.```
is thrown.

Issue Number: N/A

## What is the new behavior?
All classes added to `DialogConfig.windowClass` will be added to host element.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngneat/dialog/src/lib/dialog.component.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export class DialogComponent implements OnInit, OnDestroy {
     this.nodes.forEach(node => host.appendChild(node));
 
     if (config.windowClass) {
-      host.classList.add(config.windowClass);
+      host.classList.add(...config.windowClass.split(/\s/).filter(x => x));
```
Comment (NetanelBasal) [inline projects/ngneat/dialog/src/lib/dialog.component.ts:77]: This will work in IE?

Comment (fschick) [reply projects/ngneat/dialog/src/lib/dialog.component.ts:77]: Well, no and yes. It doesn't work natively. IE isn't supported by project's [browserlist](https://github.com/ngneat/dialog/blob/master/browserslist). If you like it work's with IE change the browserlist and required ployfills should be added automatically. Still works (and tested via [Virtual Machines](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)) with Edge 18 (supported by the browserlist)

Comment (NetanelBasal) [reply projects/ngneat/dialog/src/lib/dialog.component.ts:77]: Let's change it to:

```ts
const classNames = config.windowClass.split(/\s/).filter(x => x);
classNames.forEach(name => host.classList.add(name))
```

Comment (fschick) [reply projects/ngneat/dialog/src/lib/dialog.component.ts]: Done, tested and pushed.

**Pull request merged**